diff -Nur enigma2/data/fonts/Makefile.am enigma2.patched/data/fonts/Makefile.am
--- enigma2/data/fonts/Makefile.am	2022-07-04 16:12:32.678438941 +0200
+++ enigma2.patched/data/fonts/Makefile.am	2022-07-04 16:13:27.485096783 +0200
@@ -14,9 +14,9 @@
 	LiberationSans-Regular.ttf \
 	wqy-microhei.ttc
 
-install-data-hook:
-	cd $(DESTDIR)$(datadir)/fonts && \
-	$(LN_S) wqy-microhei.ttc fallback.font
+#install-data-hook:
+#	cd $(DESTDIR)$(datadir)/fonts && \
+#	$(LN_S) wqy-microhei.ttc fallback.font
 
 if VUULTIMO
 dist_install_DATA += analog.ttf MyriadPro-Regular.otf MyriadPro-Semibold.otf MyriadPro-SemiboldIt.otf

